src/features/home/events/events.test.tsx

```diff
@@ -24,23 +24,23 @@ describe('Events', () => {
 
   it('renders the paragraph text correctly', () => {
     const { getByText } = renderWithRouter(<Events />);
-    const paragraphText = getByText(/During 9 years/i);
+    const paragraphText = getByText(/we have organized 150\+ events/i);
     expect(paragraphText).toBeVisible();
   });
 
-  it('renders two event cards', () => {
-    const { getAllByRole } = renderWithRouter(<Events />);
-    const eventCards = getAllByRole('link', { name: /View details/i });
-    expect(eventCards.length).toBe(2);
+  it('renders two or less event cards', () => {
```

Comment:
Please add test for we can see picture (if there are no events)

Reply:
Done